### PR TITLE
all template and a json requests should be strictly enforcing a logge…

### PR DESCRIPTION
**Pull request by fredkingham**
…d in user

### CODE REVIEW

File: opal/core/api.py [hunk 1]
```diff
@@ -371,16 +373,3 @@ def post(self, *args, **kwargs):
             episode.set_tag_names(current_tags, None)
         resp = {'ok': 'Got your referral just fine - thanks!'}
         return _build_json_response(resp)
-
-
-class EpisodeListApi(View):
-    """
```
Comment (davidmiller) [inline opal/core/api.py]: @fredkingham Do you think this is not used anywhere (I can completely believe that it's not, just haven't checked myself...)

**Comment by davidmiller** [comment]
In general this looks great - I reckon it needs entries in  

* Changelog
* Upgrading (Suggest that applications or plugins defining APIs swap out Viewset for Loginrequired)
* http://opal.openhealthcare.org.uk/docs/guides/plugins/
* http://opal.openhealthcare.org.uk/docs/guides/json_api/

File: opal/core/api.py (hunk 1 above)
Comment (davidmiller) [inline opal/core/api.py]: It's not even wired up to a URL so I assume we're fine.

**Comment by fredkingham** [comment]
documentation added

**Pull request merged**